### ISSUE TITLE
crash report. Souvenirs wont open

### ISSUE THREAD

**Issue by jkbentley4** [zorgluf/souvenirs-android#26]
```
type: crash
osVersion: google/raven/raven:14/AP1A.240405.002/2024040900:user/release-keys
package: fr.nuage.souvenirs:32
process: fr.nuage.souvenirs
processUptime: 387 + 212 ms
installer: org.fdroid.fdroid

java.lang.RuntimeException: Unable to start activity ComponentInfo{fr.nuage.souvenirs/fr.nuage.souvenirs.AlbumListActivity}: java.lang.IllegalArgumentException: Shortcut must have a non-empty label
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3957)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4137)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:108)
	at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:195)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:157)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:90)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2551)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8532)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ExecInit.main(ExecInit.java:50)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:359)
Caused by: java.lang.IllegalArgumentException: Shortcut must have a non-empty label
	at androidx.core.content.pm.ShortcutInfoCompat$Builder.build(ShortcutInfoCompat.java:963)
	at fr.nuage.souvenirs.AlbumListActivity.updateShorcuts(AlbumListActivity.java:100)
	at fr.nuage.souvenirs.AlbumListActivity.onCreate(AlbumListActivity.java:65)
	at android.app.Activity.performCreate(Activity.java:8820)
	at android.app.Activity.performCreate(Activity.java:8798)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1524)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3939)
	... 15 more
```

**Comment by zorgluf** [comment]
Thanks for the report. This should be due to an empty name in the album. Fixed in v2.8.8, just released, but a few days delay if you use f-droid.
In the meantime, if you have access to the private folder of the app (dont remember if you still can do it manually with latest android), a deletion of the last album should do the remediation.
Sorry for this obvious bug...

**Comment by jkbentley4** [comment]
Thank you! I am glad you were able to fix it! I am not able to open souvenirs still so I will wait for f-droid.
I love using Souvenirs. I use it almost every day.

-------- Original Message --------
